tooling-server-configuration/src/main/java/org/jboss/eap/qe/microprofile/tooling/server/configuration/ConfigUtils.java

```diff
@@ -0,0 +1,127 @@
+package org.jboss.eap.qe.microprofile.tooling.server.configuration;
+
+import org.jboss.arquillian.config.descriptor.api.ArquillianDescriptor;
+import org.jboss.arquillian.config.descriptor.api.ContainerDef;
+import org.jboss.arquillian.core.api.annotation.Observes;
+import org.jboss.arquillian.test.spi.event.suite.BeforeSuite;
+
+import java.util.Optional;
+
+/**
+ * Provides access to configuration properties
+ */
+public class ConfigUtils {
+
+    public static final String DEFAULT_CONTAINER_NAME = "jboss";
+    public static final String DEFAULT_MANAGEMENT_ADDRESS_VALUE = "127.0.0.1";
+    public static final String DEFAULT_MANAGEMENT_PORT_VALUE = "9990";
+    public static final String ARQ_MANAGEMENT_ADDRESS_PROPERTY_NAME = "managementAddress";
+    public static final String ARQ_MANAGEMENT_PORT_PROPERTY_NAME = "managementPort";
+
+    private ConfigUtils() {}
+
+    /**
+     * This property is initialized during Before* phase by ArquillianConfiguration extension
+     */
+    private static ArquillianDescriptor arquillianDescriptor;
+
+    /**
+     * Retrieves definition for the given container name
```

Comment:
When talking about container in Arquillian context I suggest to mention it as an "Arquillian container" to differentiate it from operating system level virtualization like Podman or Docker.

Reply:
Done